source/includes/administration/_pricings.md

```diff
@@ -152,6 +155,7 @@ Attributes | &nbsp;
 `name`<br/>*Map* | A map from language to name value for that language
 `description`<br/>*Map* | A map from language to description value for that language
 `supportedCurrencies`<br/>*List* | A list of currencies supported by the pricing
+`missingCurrenciesPricing`<br/>*boolean* | Specifies if one of the change is missing the prices for one or more currencies.
```

Comment:
```suggestion
`missingCurrenciesPricing`<br/>*boolean* | Specifies if one of the changes is missing the prices for one or more currencies.
```

Reply:
Should also correct other instances

Reply:
dc3d8906541b261c5eb88057b59ca74f4d619fb1